Player/Player.tscn

```diff
@@ -209,7 +209,7 @@ position = Vector2( 0, -24 )
 texture = ExtResource( 6 )
 hframes = 9
 vframes = 13
-frame = 97
+frame = 98
```

Comment:
lol I hate these VC changes. Wonder if we can have like gitignore properties for Nodes.

Reply:
That would be nice. We can't ignore a .tscn file entirely though because we may add new nodes that need to be tracked.

Reply:
Yeah :( it seems like we'd have to somehow tell Godot to stop tracking certain things in the version control tscn file. Oh well 🤷